ambassador/ambassador/config/config.py

```diff
@@ -372,19 +385,25 @@ def validate_object(self, resource: ACResource) -> RichStatus:
             apiVersion = apiVersion.replace('getambassador.io/', 'ambassador/')
             resource.apiVersion = apiVersion
 
+        is_ambassador = False
+
         # Ditch the leading ambassador/ that really needs to be there.
         if apiVersion.startswith("ambassador/"):
+            is_ambassador = True
             apiVersion = apiVersion.split('/')[1]
+        elif apiVersion.startswith('serving.knative.dev') or apiVersion.startswith('networking.internal.knative.dev'):
```

Comment:
Should this be `elif apiVersion.endswith('.knative.dev')` ?

Reply:
We could do that, but I wanted to explicit to parse only ClusterIngress or kservice here and no other Knative resource since Ambassador does not understand anything else

Reply:
Good enough.